src/content/cookbook/testing/integration/introduction.md

```diff
@@ -145,76 +189,96 @@ counter_app/
     app_test.dart
 ```
 
-## 4. Write the integration test
-
-Now you can write tests. This involves three steps:
-
-  1. Initialize `IntegrationTestWidgetsFlutterBinding`, a singleton service that
-     executes tests on a physical device.
-  2. Interact and tests widgets using the `WidgetTester` class.
-  3. Test the important scenarios.
-
-<?code-excerpt "integration_test/app_test.dart (IntegrationTest)"?>
-```dart
-import 'package:flutter/material.dart';
-import 'package:flutter_test/flutter_test.dart';
-import 'package:integration_test/integration_test.dart';
-import 'package:introduction/main.dart';
-
-void main() {
-  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
-
-  group('end-to-end test', () {
-    testWidgets('tap on the floating action button, verify counter',
-        (tester) async {
-      // Load app widget.
-      await tester.pumpWidget(const MyApp());
-
-      // Verify the counter starts at 0.
-      expect(find.text('0'), findsOneWidget);
-
-      // Finds the floating action button to tap on.
-      final fab = find.byKey(const Key('increment'));
-
-      // Emulate a tap on the floating action button.
-      await tester.tap(fab);
-
-      // Trigger a frame.
-      await tester.pumpAndSettle();
-
-      // Verify the counter increments by 1.
-      expect(find.text('1'), findsOneWidget);
-    });
-  });
-}
-```
-
-## 5. Run the integration test
-
-The process of running the integration tests varies depending on the platform
-you are testing against. You can test against a mobile platform or the web.
-
-### 5a. Mobile
-
-To test on a real iOS / Android device, first connect the device and run the
-following command from the root of the project:
-
-```console
-$ flutter test integration_test/app_test.dart
-```
-
-Or, you can specify the directory to run all integration tests:
-
-```console
-$ flutter test integration_test
-```
- 
-This command runs the app and integration tests on the target device. For more
-information, see the [Integration testing][] page.
+## Write the integration test
+
+1. Open your `integration_test/app_test.dart` file in your preferred IDE.
```

Comment:
This is per [Google Style](https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements).

Reply:
I completely agree with Andrew here. In general, I try to avoid "your". I've been told that it can really mess up translation. Also, this sentence uses "your" TWICE, which I try to avoid.